extensions/google-meet/src/utils/scripts.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Get all tabs urls based on the browser
+ *
+ * @param browserName selected browser name
+ *
+ * @returns all tabs urls
+ */
+export const getOpenedUrlsScript = (browserName: string): string => `
+  set titleString to ""
+  
+  tell application "${browserName}"
+    set window_list to every window
+
+    repeat with the_window in window_list
+      set tab_list to every tab in the_window
+      repeat with the_tab in tab_list
+        set the_url to the URL of the_tab
+        set titleString to titleString & the_url & ","
+      end repeat
+    end repeat
+
+    return titleString
+
+  end tell
+`;
+
+// Easy way to access the focused window when the meet link opens
+export const getOpenedBrowserScript = `
+    set cmd to "lsappinfo metainfo | grep -E -o 'Google Chrome|Safari|Mozilla Firefox|Microsoft Edge|Opera|QQ|Sogou Explorer|Yandex|Brave' | head -1"
```

Comment:
@pernielsentikaer you should be able to add "Arc" here

Reply:
That found the browser indeed, but it's failing to run the script
```
ready -  built extension successfully
10:38:43.245 
Error: Command failed with exit code 1: osascript -e 
  set titleString to ""
  
  tell application "Arc"
    set window_list to every window

    repeat with the_window in window_list
      set tab_list to every tab in the_window
      repeat with the_tab in tab_list
        set the_url to the URL of the_tab
        set titleString to titleString & the_url & ","
      end repeat
    end repeat

    return titleString

  end tell

154:163: execution error: Arc got an error: Can’t make «class
```

I'm not that much into how it works with Arc yet to be able to fix it myself at least

Reply:
well, that's really a corner case... I can't access Arc because I don't have an invitation so I can't reproduce it here, it's probably related to Arc

@thomaspaulmann what do you think about something like a blacklist/skiplist, where I could set some unusual browsers and if the defaultBrowser is some of them I just skip coping the link to the clipboard, that way the meet extension keeps working like before for everyone. If that's a way I could be responsible for updating this 'blacklist' on some regularly basis (that also depends on not being privite, etc)

Reply:
well, that's really a corner case... I can't access Arc because I don't have an invitation so I can't reproduce it here, it's probably related to Arc itself

@thomaspaulmann that was part of my bad because I didn't catch the error while executing the command, I will fix it and the extension should be working as usual, but will not copy the meet link to the clipboard, do you think that it's ok to do something like

```javascript
...
  } catch (error) {
    await showHUD("Couldn't copy link to clipboard");
  }
```

or should I just do literally nothing about it?